ebos/ecloutputblackoilmodule.hh

```diff
@@ -685,6 +688,98 @@ public:
                         = tracerModel.tracerConcentration(tracerIdx, globalDofIdx);
                 }
             }
+
+            // flows
+            if (!problem.model().linearizer().getFlowsInfo().empty()){
+                const auto& flowsInf = problem.model().linearizer().getFlowsInfo();
+                auto flowsInfos = flowsInf[globalDofIdx];
+                for (auto& flowsInfo : flowsInfos) {
+                    if (flowsInfo.faceId == 1){
+                        if (FluidSystem::phaseIsActive(gasPhaseIdx) & !this->flowsi_[gasCompIdx].empty())
+                            this->flowsi_[gasCompIdx][globalDofIdx] = flowsInfo.flow[Indices::canonicalToActiveComponentIndex(gasCompIdx)];
```

Comment:
Please break lines that get very long. Or store the Indices:: canonicalToActiveComponentIndex(gasCompIdx) et. numbers in variables to get a more readable line.